packages/paste-website/src/pages/components/separator/index.mdx

```diff
@@ -0,0 +1,151 @@
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import Changelog from '@twilio-paste/separator/CHANGELOG.md';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {Table, Thead, Tbody, Tr, Td, Th} from '../../../components/table';
+import {InlineCode} from '../../../components/Typography';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/separator"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/separator/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Separator"
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/separator"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-separator--horizontal"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+### About Separator
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+It can be used to create separation between large sections of page content, or between smaller parts of single UI controls such as menu items in a menu.
+
+## Examples
+
+### Vertical separation
+
+Vertical separation is achieved by setting the `orientation` property to horizontal. A `verticalSpacing` styling property is provided to enable you to create space above and below the separator. `verticalSpacing` will take any [Spacing Token](/tokens#spacings) as it's value.
+
+<LivePreview scope={{Separator}} language="jsx">
+  {`<Separator orientation="horizontal" verticalSpacing="space50" />`}
+</LivePreview>
+
+### Horizontal separation
+
+<LivePreview scope={{Separator, Text, Box}} language="jsx">
+  {`<Box display="flex">
+  <Text as="span">Option 1</Text>
+  <Separator orientation="vertical" horizontalSpacing="space40" />
+  <Text as="span">Option 2</Text>
+  <Separator orientation="vertical" horizontalSpacing="space40" />
+  <Text as="span">Option 3</Text>
+  <Separator orientation="vertical" horizontalSpacing="space40" />
+  <Text as="span">Option 4</Text>
+  <Separator orientation="vertical" horizontalSpacing="space40" />
+  <Text as="span">Option 5</Text>
+</Box>`}
+</LivePreview>
+
+### Spacing
+
+The Separator comes with 2 spacing properties; `horizontalSpacing` and `verticalSpacing`. You can use these properties to create space between the Separator and the two pieces of UI it sits between.
+
+When used the spacing properties create equal space in both directions along the orientation you choose.
```

Comment:
Nice.

- "used" --> "using"
- add comma after "properties"

Reply:
Shouldn't it be?

`When used, the spacing properties create equal space in both directions along the orientation you choose`

Reply:
ohhhh ok I read that wrong. Could prob just remove "when used".